libcodechecker/analyze/analyzers/analyzer_types.py

```diff
@@ -214,11 +212,27 @@ def initialize_checkers(config_handler,
 
     # Set user defined enabled or disabled checkers from the command line.
     if cmdline_checkers:
-        for checker_name, enabled in cmdline_checkers:
-            if enabled:
-                config_handler.enable_checker(checker_name)
+        for identifier, enabled in cmdline_checkers:
```

Comment:
Maybe we should diagnose somewhere if a profile name clashes with a checker name or "list". We should issue an error in that case.